plugins/setup.cfg

```
@@ -8,9 +8,9 @@ classifiers =
 packages = find:
 include_package_data = True
 install_requires =
-    napari
+    napari[all]
     qtpy
 
 [options.entry_points]
 napari.manifest =
-    napari-ADS = plugins.napari_ADS:napari.yaml
\ No newline at end of file
+    napari-ADS = plugins.napari_ADS:napari.yaml

```

```suggestion
    napari-ADS = napari_ADS:napari.yaml

[options.package_data]
napari_ADS = napari.yaml
```

Explanation: 

- On Windows, if `plugins.` is present, then Napari will try to import from a package named `plugins` (which doesn't exist). Removing this will allow Napari to import from `napari_ADS` instead.
- However, removing `plugins.` causes Napari to be unable to find the `napari.yaml` on Linux (weird!). To fix this, we add a new entry to `setup.cfg` to specify that the Napari Manifest file should be included as package data when `napari_ADS` is installed. (Taken directly from [Napari documentation](https://napari.org/0.4.15/plugins/manifest.html#manifest-reference).)

When these lines are added, the plugin loads correctly on both Windows and Linux.

---

Note: `plugins.` may also need to be removed from the `napari.yaml` file, too.